src/app/child-dev-project/children/model/child.ts

```diff
@@ -129,7 +129,11 @@ export class Child extends Entity {
 
   get isActive(): boolean {
     return (
-      this.status !== "Dropout" && !this["dropoutDate"] && !this["exit_date"]
+      this.status !== "Dropout" &&
+      !this["dropoutDate"] &&
+      !this["exit_date"] &&
+      !(this["inactive"] !== undefined && this["inactive"] === true) &&
+      !(this["active"] !== undefined && this["active"] === false)
```

Comment:
```suggestion
      !this["exit_date"] &&
      this.super.isActive;
```
Lets try to make this less complex. It took me a few minutes to figure out the branches that are possible here.

Reply:
nice :+1: